app/femr/ui/views/admin/configure/manage.scala.html

```diff
@@ -13,13 +13,14 @@
             <tr>
                 <th>Feature</th>
                 <th>Toggle</th>
+                <th>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Description</th>
             </tr>
 
             @for((key, valyew) <- viewModel.getSettings) {
                 <tr>
                     <td class="name">@key</td>
                     <td class="isActive">
-                        <input type="checkbox" name="settings[]" value="@key" @if(valyew) { checked="checked"} />
+                        <input type="checkbox" name="settings[]" value="@key" @if(valyew) { checked="checked"} >
                     </td>
```

Comment:
Hi Ken,
Thanks for the review. I will make the required changes and submit again. I
added the "/" intentionally to the line 23 as it was showing an error, I
wasn't knowing that it would create other problems. I would remove it
anyways.

Thanks again,
Kratika
On 2 Mar 2016 8:36 pm, "Ken Dunlap" <notifications@github.com> wrote:

> In app/femr/ui/views/admin/configure/manage.scala.html
> <https://github.com/FEMR/femr/pull/487#discussion_r54824230>:
>
> >              </tr>
> >
> >              @for((key, valyew) <- viewModel.getSettings) {
> >                  <tr>
> >                      <td class="name">@key</td>
> >                      <td class="isActive">
> > -                        <input type="checkbox" name="settings[]" value="@key" @if(valyew) { checked="checked"} />
> > +                        <input type="checkbox" name="settings[]" value="@key" @if(valyew) { checked="checked"} >
> >                      </td>
>
> Maybe this was taken out by accident, but the closing slash makes that
> input field valid XHTML. This is not the end of the world, but could you
> add that back in? Line 23 should read:
>
> <input type="checkbox" name="settings[]" value="@key" @if(valyew) {
> checked="checked"} />
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FEMR/femr/pull/487/files#r54824230>.
>